var/spack/repos/builtin/packages/py-fenics-dijitso/package.py

```diff
@@ -15,11 +15,11 @@ class PyFenicsDijitso(PythonPackage):
     maintainers = ["js947", "chrisrichardson"]
 
     version("master", branch="master")
-    version("2018.1.0", sha256="2084ada1e7bd6ecec0999b15a17db98c72e26f1ccbf3fcbe240b1a035a1a2e64")
-    version("2017.2.0", sha256="05a893d17f8a50067d303b232e41592dce2840d6499677ad8b457ba58a679b58")
-    version("2017.1.0", sha256="ef23952539d349fbd384d41302abc94413d485ca7e8c29e8a0debc2aadaf8657")
-    version("2016.2.0", sha256="1bfa0ac0d47dae75bbde8fabb1145d3caa2e6c28a1e4097ad5550a91a8a205e4")
+    version('2019.1.0', sha256='eaa45eec4457f3f865d72a926b7cba86df089410e78de04cd89b15bb405e8fd9')
+    version('2018.1.0', sha256='2084ada1e7bd6ecec0999b15a17db98c72e26f1ccbf3fcbe240b1a035a1a2e64')
+    version('2017.2.0', sha256='05a893d17f8a50067d303b232e41592dce2840d6499677ad8b457ba58a679b58')
+    version('2017.1.0', sha256='ef23952539d349fbd384d41302abc94413d485ca7e8c29e8a0debc2aadaf8657')
+    version('2016.2.0', sha256='7358c6cbeacaada3ef3071bdcf2a5f342cb7ee1ed736742d2b747b1fd4454d4c')
```

Comment:
Why did this checksum change?

Reply:
The checksums were updated based on the result from the `spack checksum` command, why they have changed, I can't tell.